boards/xtensa/esp32s_nodemcu/doc/index.rst

```diff
@@ -0,0 +1,220 @@
+.. _esp32:
+
+ESP32
+#####
+
+Overview
+********
+
+ESP32 is a series of low cost, low power system on a chip microcontrollers
+with integrated Wi-Fi & dual-mode Bluetooth.  The ESP32S NodeMCU employs a
+Tensilica Xtensa LX6 dual-core microprocessor.  ESP32 is created and
+developed by Espressif Systems, a Shanghai-based Chinese company,
+and is manufactured by TSMC using their 40nm
+process. [1]_
+
+The features include the following:
+
+- Dual core Xtensa microprocessor (LX6), running at 240MHz
+- 520KB of SRAM
+- 802.11b/g/n/e/i
+- Bluetooth v4.2 BR/EDR and BLE
+- Various peripherals:
```

Comment:
Please check this information against the esp32s2 reference manual, I'm afraid this is not correct